### ISSUE TITLE
Feature request: Add a11y testing

### ISSUE THREAD

**Issue by mejiaj** [Bixal/bx-nextjs-storybook-uswds#6]
## Summary

Add automated a11y testing in storybook and nextjs app.

## Alternatives you've considered

Manual testing, but it can be inconsistent and easy to forget.

## Additional context

Will need avoid adding too many dependencies. Whatever we test should be against section 508 and WCAG 2.2 AA.
